specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-04-02/storagesync.json

```diff
@@ -2508,6 +2535,210 @@
+      },
+      "description": "The parameters used when creating a storage sync service."
+    },
+    "SyncGroupCreateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update, the request will succeed."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SyncGroupCreateParametersProperties",
+          "description": "The parameters used to create the sync group"
+        }
+      },
+      "description": "The parameters used when creating a sync group."
+    },
+    "SyncGroupCreateParametersProperties": {
+      "type": "object",
+      "description": "Sync Group Create Properties object.",
+      "properties": {
+       }
+    },
+    "CloudEndpointCreateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update, the request will succeed."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used for viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key with a length no greater than 128 characters and a value with a length no greater than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudEndpointCreateParametersProperties",
+          "description": "The parameters used to create the storage sync service."
+        }
+      },
+      "description": "The parameters used when creating a storage sync service."
+    },
+    "CloudEndpointCreateParametersProperties": {
+      "description": "CloudEndpoint Properties object.",
+      "properties": {
```

Comment:
Example: This is lacking a lot of the properties that were previously available in "CloudEndpointProperties".

Reply:
There is NO need of version bump as these changes fixed the swagger spec and make it consistent with what's in the PROD. CSharp verification confirms that.

Also, These changes are aligned with other resource provider has : CreateParameters and UpdateParameters.

Just to bring clarity, There is NO SDK/AzureRM Powershell for existing Swagger spec , which gives us a flexibility to fix the mistakes with in the same version.

Please signoff and unblock us. Hope it helps.

Reply:
If there are truly no changes to the service behind this, and it is just aligning swagger to the existing API, I will sign-off from ARM's perspective.